src/services/legal.services.ts

```diff
@@ -186,4 +186,28 @@ export default class LegalServices {
     }
     return toReturn
   }
+
+  /**
+   * Fetches parties list.
+   * @param businessId the business identifier (aka entity inc no)
+   * @returns a promise to return the parties from the response
+   */
+  static fetchParties (businessId: string): Promise<any> {
+    // need to get the parties from the legal endpoint v2
+    const legalApiV2 = sessionStorage.getItem('LEGAL_API_URL_V2')
+    let url = `${legalApiV2}businesses/${businessId}/parties`
+
+    return axios.get(url).then(response => {
+      const data = response?.data
+      if (!data) {
+        throw new Error('Invalid API response')
+      }
+      return data
+    }).catch(error => {
+      if (error?.response?.status === NOT_FOUND) {
+        return null // parties not found
```

Comment:
It looks like you copied this from fetchNameRequest(). That handling allows for "not found" as a valid scenario.

It's not a valid scenario here, so I would simplify the error handling: I think you can omit the catch handler altogether. Example: https://github.com/bcgov/business-edit-ui/blob/0dd1590dc82f7f29c83dbcf41c80b807d7144e6b/src/services/auth-services.ts#L45